core/__tests__/actions/destinations.ts

```diff
@@ -514,7 +513,12 @@ describe("actions/destinations", () => {
           })
         );
 
-        await specHelper.runAction("destination:unTrackGroup", connection);
+        connection.params = {
+          csrfToken,
+          id,
+          trackedGroupId: "_none",
+        };
+        await specHelper.runAction("destination:edit", connection);
```

Comment:
Might want to confirm here that the destination you get back from `runAction` (or the model directly) isn't tracking a group any more